app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Metrics.kt

```diff
@@ -9,4 +12,9 @@ class Metrics {
     val usage: List<SkillData> = ArrayList()
     val rating: List<SkillData> = ArrayList()
     val latest: List<SkillData> = ArrayList()
+    val newest: List<SkillData> = ArrayList()
+
+    @SerializedName("Games, Trivia and Accessories")
```

Comment:
This is the field name, seriously?

Reply:
Yes, for now it is because of implementation in backend. :(

```
"Games, Trivia and Accessories": [
      {
        "model": "general",
        "group": "Games, Trivia and Accessories",
        "language": "en",
        "developer_privacy_policy": null,
        "descriptions": "Play with words, ask SUSI to give words beginning and ending with a particular alphabet.",
        "image": "images/playing_with_words.png",
        "author": "Dravit Lochan",
        "author_url": "https://github.com/DravitLochan",
        "author_email": null,
        "skill_name": "Playing With Words",
        "protected": false,
        "reviewed": false,
        "editable": true,
        "staffPick": false,
        "terms_of_use": null,
        "dynamic_content": true,
        "examples": ["word starting with m and ending with v"],
        "skill_rating": {
          "negative": "0",
          "bookmark_count": 0,
          "positive": "1",
          "stars": {
            "one_star": 0,
            "four_star": 3,
            "five_star": 4,
            "total_star": 7,
            "three_star": 0,
            "avg_star": 4.57,
            "two_star": 0
          },
          "feedback_count": 3
        },
        "usage_count": 22,
        "skill_tag": "playing_with_words",
        "supported_languages": [{
          "name": "playing_with_words",
          "language": "en"
        }],
        "creationTime": "2018-07-19T19:19:32Z",
        "lastAccessTime": "2018-07-27T22:14:41Z",
        "lastModifiedTime": "2018-07-19T19:19:32Z"
      },
```

Reply:
@iamareebjamal I am sending a PR on server to correct this.

Reply:
@iamareebjamal I checked the code on susi_server, the API is designed in such a way that it takes the group name as the key name. I am not sure if changing the keys would be the right thing to do as of now. What do you suggest?